main.go

```diff
@@ -69,8 +65,10 @@ func main() {
 	}
 
 	reload := time.NewTicker(time.Second * time.Duration(config.RefreshSeconds))
-	timeout := time.AfterFunc(zkInitialDetectionTimeout, func() {
-		errch <- fmt.Errorf("master detection timed out after %s", zkInitialDetectionTimeout)
+
+	zkTimeout := time.Second * time.Duration(config.ZkTimeout)
```

Comment:
With my upcoming flag config change, all of these durations will be of type `time.Duration`. :-)